docs/beman-standard.md

```diff
@@ -136,6 +135,28 @@ following example's style:
 
 ## `CMakeLists.txt`
 
+**[CMAKE.DEFAULT]** RECOMMENDATION: The root `CMakeLists.txt` should build all targets by default (including dependency targets).
+
+**[CMAKE.SKIP_TESTS]** RECOMMENDATION: The root `CMakeLists.txt` should not build tests and their dependencies when `BUILD_TESTING` is set to `OFF` (see [CTest docs](https://cmake.org/cmake/help/latest/module/CTest.html)). Use the following style:
+
+```CMake
+# <repo>/CMakeLists.txt
+# ...
+if(BUILD_TESTING)
+  FetchContent_Declare(
+    googletest
+    GIT_REPOSITORY https://github.com/google/googletest.git
+    GIT_TAG f8d7d77c06936315286eb55f8de22cd23c188571 # release-1.14.0
+  )
+  FetchContent_MakeAvailable(googletest)
+endif()
+
+# ...
+if(BUILD_TESTING)
+  add_subdirectory(/path/to/tests) # CMAKE.AVOID_PASSTHROUGHS already applied.
```

Comment:
```suggestion
  add_subdirectory(/path/to/tests)
```

Suggest removing the comment since it doesn't apply to this entry and may confuse folks.